### ISSUE TITLE
Issue für schema.org schreiben

### ISSUE THREAD

**Issue by jaFHP** [McNamara84/gfz-metadata-editor-msl-v2#362]
Mithilfe von Alis Tabelle entscheiden, was wir umsetzen und ein Issue schreiben

**Comment by McNamara84** [comment]
I would like to propose that we close this issue again. As we have decided that, in the long term, we will no longer save or export data via the database (Input Fields → Database → FreeStyle XML → DataCite XML), but directly via XSLT (Input Fields → DataCite XML), this would be an unnecessary step involving additional effort. In future, only vocabularies will be stored in the database, and these will follow the naming conventions specified by DataCite.

Am I thinking right?

**Comment by Ali-GeoFZ** [comment]
Yes, you’re absolutely right. I can also see another reason to close this issue, namely that the analysis was carried out two years ago, a lot has changed in our project structure since then. 

For these reasons, I am closing this issue.
